ceph-releases/kraken/ubuntu/16.04/daemon/common_functions.sh

```diff
@@ -39,27 +39,45 @@ function prefix_length {
   done
 }
 
-# create socket directory
-function create_socket_dir {
+
+# create the mandatory directories
+function create_mandatory_directories{
```

Comment:
you need a space after `create_mandatory_directories`. Otherwise the function is not happy.